ssl/statem/extensions_clnt.c

```diff
@@ -538,7 +544,7 @@ static int add_key_share(SSL *s, WPACKET *pkt, unsigned int curve_id)
         assert(s->hello_retry_request);
         if (!s->hello_retry_request) {
             SSLerr(SSL_F_ADD_KEY_SHARE, ERR_R_INTERNAL_ERROR);
-            return 0;
+            return EXT_RETURN_FAIL;
```

Comment:
Hmm, the signature of various ```static int foo(.)``` helpers was not changed, though the return statements now use the enum.  Do we care?